trin-core/Cargo.toml

```diff
@@ -77,7 +74,8 @@ version = "0.26.3"
 features = ["bundled"]
 
 [dev-dependencies]
+env_logger = "*"
```

Comment:
Why the wildcard here and elsewhere? I would prefer a more granular version.

Reply:
agreed, this allows for any version (not just latest), which might bring unexpected changes

Reply:
> Why the wildcard here and elsewhere? I would prefer a more granular version.

It is because `env_logger` is required only as a dependency of `test-log` and we don't really care about the exact version, but I agree it is more convenient to set a fixed version number.